year-pages/2019-conferences-list.md

```diff
@@ -3,4 +3,5 @@
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
 | [RubyConf India](https://www.rubyconfindia.org/) | 20-21 Jan | Resort Rio and Club Rio Royale, Goa | A global event complementing other RubyConf events across the world. This year it's a single track 2 day event, focused on the Language, Framework and Tools. | [Yes](https://www.rubyconfindia.org/scholarship/) |
-| [JSFoo](https://jsfoo.in/2019-pune/) | 10 Jan | MCCIA Trade Tower, Pune | The second Pune edition of India’s premier JavaScript conference. | No
\ No newline at end of file
+| [JSFoo](https://jsfoo.in/2019-pune/) | 10 Jan | MCCIA Trade Tower, Pune | The second Pune edition of India’s premier JavaScript conference. | No
+| [GopherCon](https://gopherconindia.com/) | 18-19 Jan | Resort Rio, Goa | The fifth Go Conference in India. | No
```

Comment:
This needs to go above JSFoo since this is from 18-19 Jan. :)

Also, gophercon india has scholarships: https://gopherconindia.com/scholarship/

Reply:
This still needs to be fixed.